snuba/admin/clickhouse/tracing.py

```diff
@@ -66,115 +29,10 @@ def run_query_and_get_trace(storage_name: str, query: str) -> TraceOutput:
     query_result = connection.execute(
         query=query, capture_trace=True, with_column_types=True
     )
-    formatted_trace_output = format_trace_output(query_result.trace_output)
+    summarized_trace_output = summarize_trace_output(query_result.trace_output)
     return TraceOutput(
         trace_output=query_result.trace_output,
-        formatted_trace_output=formatted_trace_output,
+        summarized_trace_output=summarized_trace_output,
```

Comment:
Wondering if this will break the frontend [here](https://github.com/getsentry/snuba/blob/master/snuba/admin/static/tracing/query_display.tsx#L57)  now that you changed the key of the `TraceOutput` object.

Reply:
Nevermind, just read your comment - https://github.com/getsentry/snuba/pull/6226#issuecomment-2299621214